### DISCUSSION TITLE
running netbox on AWS with ECS and RDS?

### DISCUSSION THREAD

**Discussion by richard063** [netbox-community/netbox#13720] [Help Wanted!]
Hello, I'd like to ask if anyone running netbox on AWS - ECS ?

if yes, may i know if there any site or knowledge base page that i can refer and read?

I can have netbox up and running on the EC2 with RDS just fine, but when i try to use ECS(ecr)  with ALB it doesn't work at all.

the part I'm lost is how am i going to integrate the nginx and netbox in ECS ?

**Comment by cregkly** [comment]
https://www.gooddata.com/blog/netbox-deployment-to-aws-in-one-click/

**Reply by sierrasoleil** [reply]
Do you have experience with this actually working? I can't get Netbox to authenticate to RDS. No matter what I try, the pods fails with "FATAL:  password authentication failed for user "netbox"" I spent most of yesterday on a support call with AWS, and they can't find any issue with my RDS setup. I can even auth to RDS using psql from a generic ubuntu pod running in the same cluster. I feel like I must be missing something basic in my netbox manifest, but I can't figure out what it is.
